stockflux-chart/src/components/Chart.js

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import bitflux from 'BitFlux/dist/bitflux';
+import './chart.css';
 
-import styles from './Chart.module.css';
+export default function Chart() {
+    let showcaseContainer;
+    const chart = bitflux.app();
+    useEffect(() => {
 
-export default () => (
-  <div className={styles.chart}>
-    Chart Not implemented
-  </div>
-);
+        chart.periodsOfDataToFetch(1200);
+        chart.proportionOfDataToDisplayByDefault(112/1200);
+        // using random generated data
+        chart.run(showcaseContainer);
```

Comment:
This will run on every render, is that correct? Moving forward we'll want to stream the data from the backend service.